code/datums/gamemodes/_game_mode.dm

```diff
@@ -45,8 +45,12 @@ GLOBAL_VAR(common_report) //Contains common part of roundend report
 	var/time_between_round = 0
 	///What factions are used in this gamemode, typically TGMC and xenos
 	var/list/factions = list(FACTION_TERRAGOV, FACTION_ALIEN)
-	///Increases the amount of xenos needed to evolve to tier three by the value.
+	///Reduces the number of T3 slots xenos get by the value.
 	var/tier_three_penalty = 0
+	///Includes T3 xenos in the calculation for maximum T3 slots, 1 for yes 0 for no.
+	var/tier_three_inclusion = 0
+	///Reduces the caste swap timer by the value.
+	var/caste_swap_timer_reduction = 0 MINUTES
```

Comment:
couldnt you just make a caste_swap_time? so it's easier to tell what the actual number is